duckduckgo/iOS
# Remove fire button experiment

Reviewer: Francesco
Asana: https://app.asana.com/0/276208855376242/290598814185289

**Description**:
A nice easy one! Removes the omnibar fire experiment from the app.

![screen shot 2017-05-31 at 21 17 11](https://cloud.githubusercontent.com/assets/1652407/26652460/e8d5a0d4-4647-11e7-825c-88df16888a9c.png)

**Steps to test this PR**:
TEST 1
- Launch the app
- Go to settings
- Note that the experiment toggle (visible in the attached image) is no longer there

TEST 2
- Launch the app
- Navigate to a webpage
- Note that pressing the fire button closes the current tab and takes you to the tab switcher

###### Reviewer Checklist:


###### PR DRI Checklist:
